hilgendorf_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,61 @@
+joint_limits:
+  right_wrist_2_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
If it was changed after the initial commit, it wasn't intentional. I've enabled joint accel limits and specified the maximum as 5 as you suggest. Will push the commit soon.